tests/qemu/linux/test_qemu_platform_detail.cpp

```diff
@@ -153,12 +153,10 @@ TEST_F(QemuPlatformDetail, platform_args_generate_net_resources_removes_works_as
     const auto platform_args = qemu_platform_detail.vm_platform_args(vm_desc);
 
     // Tests the order and correctness of the arguments returned
-    std::vector<QString> expected_platform_args{
-        "--enable-kvm", "-cpu", "host", "-nic",
-        QString::fromStdString(fmt::format("tap,ifname={},script=no,downscript=no,model=virtio-net-pci,mac={}",
-                                           tap_name, vm_desc.default_mac_address))};
-
-    EXPECT_THAT(platform_args, ElementsAreArray(expected_platform_args));
+    EXPECT_THAT(platform_args, ElementsAre("--enable-kvm", "-bios", _, "-cpu", "host", "-nic",
```

Comment:
Nice, but can you test that the wildcard parameter is the expected one on each platform?